Acceptable Usage Policy.md

```diff
@@ -1,59 +1,149 @@
+
+  An exception is made in regards to violence depicted in media (within the context of journalism, movies, games, music, books, and such).
+
+- **Distributing malware**
+
+  Any software designed with malicious intent to attack another person or steal their data is not tolerated.
+
+- **Impersonation of individuals, groups, etc with intent to cause harm or deception**
+
+  Pretending to be another person, organization, or entity in a way that is intended to mislead or harm others is prohibited.
+
+- **As a minor, engaging in any conduct that puts your safety at risk**
+
+  Any sexual conduct involving minors, whether online or offline, is strictly prohibited and will be reported to the appropriate authorities. We prioritize the safety of our younger users and prohibit any behaviour that could compromise their well-being.
+
+- **Sharing sexually explicit content in places where it can't be age restricted**
+
+  Sexually explicit content must be properly age-gated and should not be shared in unrestricted areas of the platform.
+
+## Keeping It Fair
+
+To ensure users have equal opportunity to use the service and are not taken advantage of, we disallow:
+
+- **Unsolicited advertising**
+
+  Creating accounts or using existing accounts to send unsolicited advertisements to other people on the platform.
+
+- **Financial scams / fraud**
+
+  Any attempt to deceive users for financial gain, including phishing, multi level marketing, or other fraudulent activities.
+
+- **Buying and selling of Revolt accounts**
+
+  The transfer of Revolt accounts for monetary or other compensation is not permitted.
+
+- **Selling, purchasing, or providing means to artificially grow servers**
+
+  This includes botting, paid advertisements, and any other methods of artificially inflating server membership or engagement. Legitimate marketing efforts and community-building strategies that don't involve artificial inflation of metrics are generally acceptable. If you're unsure whether a growth strategy violates this policy, please contact our support team for clarification.
+
+- **Evasion of permanent platform level moderation actions**
+
+  Attempting to circumvent bans or other moderation actions by creating new accounts or using alternative methods to access the platform is prohibited.
+
+- **Engaging in hacking**
+
+  Any attempt to compromise the security of our platform or its users, including phishing attacks, distribution or creation of malware, or Denial of Service attacks, is strictly forbidden.
+
+- **Attempts to hack or otherwise gain unauthorised access to our service**
+
+  For example, accessing another person's account without permission, or attempting to steal another person's account. This also includes attempts to gain access to our backend networks. _Please contact us ahead of time for permission to test our networks by emailing [security@revolt.chat](mailto:security@revolt.chat)._
```

Comment:
isnt this pretty much the same thing as the above, with different wording?

Reply:
Oh I suppose it is.